client/i18n/locales/english/trending.json

```diff
@@ -1,62 +1,62 @@
-  "article13link": "https://www.freecodecamp.org/news/javascript-string-replace-example-with-regex/",
-  "article14title": "Python JSON Parser",
-  "article14link": "https://www.freecodecamp.org/news/python-read-json-file-how-to-load-json-from-a-file-and-parse-dumps/",
-  "article15title": "cmd Delete Folder",
-  "article15link": "https://www.freecodecamp.org/news/cmd-delete-folder-how-to-remove-files-and-folders-in-windows/",
-  "article16title": "What is NFC?",
-  "article16link": "https://www.freecodecamp.org/news/what-is-nfc-near-field-communication-uses-chips-tags-and-readers-explained/",
-  "article17title": "Content Type JSON",
-  "article17link": "https://www.freecodecamp.org/news/what-is-the-correct-content-type-for-json-request-header-mime-type-explained/",
-  "article18title": "Convert HEIC to JPG",
-  "article18link": "https://www.freecodecamp.org/news/heic-to-jpg-how-to-convert-images-on-a-mac/",
-  "article19title": "Math Random Java",
-  "article19link": "https://www.freecodecamp.org/news/generate-random-numbers-java/",
-  "article20title": "WordPress for Beginners",
-  "article20link": "https://www.freecodecamp.org/news/how-to-start-a-blog-with-wordpress/",
-  "article21title": "Qualitative VS Quantitative",
-  "article21link": "https://www.freecodecamp.org/news/qualitative-vs-quantitative-definition-research-methods-and-data/",
-  "article22title": "JavaScript Split String",
-  "article22link": "https://www.freecodecamp.org/news/javascript-split-string-example/",
-  "article23title": "Accented Letters on Mac",
-  "article23link": "https://www.freecodecamp.org/news/how-to-type-letters-with-accents-on-mac/",
-  "article24title": "Windows 10 Product Key",
-  "article24link": "https://www.freecodecamp.org/news/how-to-find-a-windows-10-product-key/",
-  "article25title": "Google Docs Landscape",
-  "article25link": "https://www.freecodecamp.org/news/google-docs-landscape-tutorial-how-to-make-a-google-doc-landscape-orientation/",
-  "article26title": "Antimalware Executable",
-  "article26link": "https://www.freecodecamp.org/news/antimalware-service-executable-what-is-msmpeng-exe-and-why-is-it-running-in-windows-10/",
-  "article27title": "Windows 10 Start Menu",
-  "article27link": "https://www.freecodecamp.org/news/windows-10-start-menu-not-working-solved/",
-  "article28title": "Windows 10 Command Line",
-  "article28link": "https://www.freecodecamp.org/news/how-to-open-the-command-prompt-in-windows-10/",
-  "article29title": "Google Account Recovery",
-  "article29link": "https://www.freecodecamp.org/news/google-account-recovery-change-gmail-password/"
+  "article0title":"10 to the Power of 0",
+  "article0link":"https://www.freecodecamp.org/news/10-to-the-power-of-0-the-zero-exponent-rule-and-the-power-of-zero-explained/",
+  "article1title":"Git Reset to Remote",
+  "article1link":"https://www.freecodecamp.org/news/git-reset-to-remote-head-how-to-reset-a-remote-branch-to-origin/",
+  "article2title":"r value statistics",
```

Comment:
```suggestion
  "article2title":"R Value in Statistics",
```

Reply:
@scissorsneedfoodtoo, should we go the capital version? the following is what was on he card.
<img width="113" alt="Screen Shot 2021-03-30 at 11 05 30 AM" src="https://user-images.githubusercontent.com/4591597/112955206-dbb96d00-9147-11eb-901d-3fa6b54d7ab0.png">

Reply:
Thanks for catching this @ShaunSHamilton. I think we did miss the title casing on the Trello card for this article.

I did a bit of searching and have also seen it as either R or r in other titles. Let's go with this suggestion to keep the casing in the footer consistent.